### ISSUE TITLE
Containers do not start automatically after Reboot

### ISSUE THREAD

**Issue by Coxeroni** [TechPerplexed/Gooby#188]
As the title says, my containers do not start automatically anymore. I updated to the most recent Gooby but it didn't help.
I have to bring them up manually each time.

Which switch do I need to flip to bring this back to live?

**Comment by TechPerplexed** [comment]
What exactly do you mean that they are not coming up automatically? Where do they get stuck?

**Comment by Coxeroni** [comment]
Sorry for being unspecific, but no container is actually starting by itself. When I check via docker ps after a reboot, there is emptiness.
When executing docker-compose up -d everything starts fine without any error.

**Comment by TechPerplexed** [comment]
I see 😊 Can you verify that rclean works or does that spot out errors as well?

**Comment by Coxeroni** [comment]
If rclean is System Cleanup in Gooby than yes there are Errors:
`cat: /var/local/Gooby/Docker/.config/rclonefolder: No such file or directory
done
Job for mergerfs.service failed because the control process exited with error code.
See "systemctl status mergerfs.service" and "journalctl -xe" for details.
Waiting on mounts to be created...
Waiting on mounts to be created...
Waiting on mounts to be created...
Waiting on mounts to be created...
Waiting on mounts to be created...
Waiting on mounts to be created...
Waiting on mounts to be created...
Waiting on mounts to be created...`

So obviously I am also missing mount points?

**Comment by TechPerplexed** [comment]
Yes, you would have... But I think that will be ready to sort... It seems that with some, the update to parameterize the mounts did not go as smooth as was hoped.

Can you please try this:

sudo systemctl daemon-reload
sudo systemctl stop mergerfs
sudo systemctl stop rclonefs
/bin/fusermount -uz /mnt/rclone
/bin/fusermount -uz /mnt/google

Verify that /mnt/google does not exist

Then run rclean again...

If that still doesn't work, try all the above and then uninstall Rclone from the Gooby menu, then reinstall it. I don't think that last step will be necessary though 😊

**Comment by Coxeroni** [comment]
I had to go all the way including the rclone reinstall, but now rclean went through without errors and I believe I am back in the game :)

Thanks for your swift help, always a pleasure!

**Comment by TechPerplexed** [comment]
Great! And sorry about the non-smooth upgrade process 😁
